packages/web-components/src/progressbar/progressbar.stories.ts

```diff
@@ -0,0 +1,83 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import type { ProgressBar as FluentProgressBar } from './progressbar.js';
+import { ProgressBarShape, ProgressBarThickness, ProgressBarValidationState } from './progressbar.options.js';
+import './define.js';
+
+type ProgressStoryArgs = Args & FluentProgressBar;
+type ProgressStoryMeta = Meta<ProgressStoryArgs>;
+
+const storyTemplate = html<ProgressStoryArgs>`
+  <fluent-progressbar
+    paused=${x => x.paused}
```

Comment:
I believe this is a boolean attribute, so we should use that binding here:
```suggestion
    ?paused=${x => x.paused}
```